flight/Modules/Battery/battery.c

```diff
@@ -100,6 +101,7 @@ static void batteryTask(void * parameters)
 	const float dT = SAMPLE_PERIOD_MS / 1000.0f;
 
 	battery_settings_updated = true;
+	cells_calculated = false;
```

Comment:
Just add bool in front of this and delete the declaration at the top to reduce scope.